assets/training/general/environments/acpt-pytorch-2.0-cuda11.7/tests/pytorch2_0_sample_test.py

```diff
@@ -12,7 +12,7 @@
 
 BUILD_CONTEXT = Path("../context")
 JOB_SOURCE_CODE = "../../acpt-tests/src"
-TIMEOUT_MINUTES = os.environ.get("timeout_minutes", 60)
+TIMEOUT_MINUTES = os.environ.get("timeout_minutes", 720)
```

Comment:
do we know why we need to increase timeout from 60 to 720?

Reply:
Oh, it timed out at 60 min on some runs for some reason, even with just the existing tests that I did not otherwise modify. I put it to 720 to avoid any timeout. I can set it back to 60 if that's preferred

Reply:
reverted to 60 min as per our offline discussion